src/app/client/src/app/app.component.ts

```diff
@@ -416,11 +416,11 @@ export class AppComponent implements OnInit, OnDestroy {
         this.orgDetails = data;
         this.channel = this.orgDetails.hashTagId;
         this.botObject['channel'] = this.channel;
-        if (this.userService.slug !== '') {
+        // if (this.userService.slug !== '') {
```

Comment:
what happen if the slug value is not there you are still saving the slug with empty string?